app/src/main/java/tech/redroma/yoching/ActivityPlus.kt

```diff
@@ -57,3 +59,37 @@ internal fun Fragment.perform(block: Fragment.() -> Unit)
 {
     block(this)
 }
+
+
+private object Icons
+{
+    lateinit var headsIcon: Drawable
+    lateinit var tailsIcon: Drawable
+    var isInitalized = false
+
+    fun initialize(context: Context)
+    {
+        if (isInitalized)
+        {
+            return
+        }
+
+        headsIcon = context.resources.getDrawable(R.drawable.coin_heads_slick)
+        tailsIcon = context.resources.getDrawable(R.drawable.coin_tails_slick)
```

Comment:
Forgot to set `isInitialized` to `true` here. 

![bitmoji](https://render.bitstrips.com/v2/cpanel/10187987-288439948_4-s1-v1.png?transparent=1&palette=1&width=246)